src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -531,6 +523,7 @@ export function getInputType(
         if (!model) {
             const baseModel = getInputModelBaseType(m.baseModel);
             const properties: InputModelProperty[] = [];
+            const derivedModels: InputModelType[] = [];
```

Comment:
`DerivedModels` is optional, can we not default set it as empty array? If no derived models, set `DerivedModels` undefined.